web/static/js/components/room.jsx

```diff
@@ -18,6 +18,13 @@ const Room = props => {
   const progressionConfig = stageProgressionConfigs[stage]
   const showActionItem = stage !== "idea-generation"
 
+  function checkForActionItems() {
```

Comment:
hmmm... the naming here can be more expressive. this may not be the right name, but something along the lines of `noActionItemsSubmitted` would be more expressive of what the method is doing